cfgov/v1/preprocessed/css/molecules/text-introduction.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Theme variables
+  family: cfgov-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @m-text-introduction_rule
+  tags:
+  - less
+*/
+@m-text-introduction_rule: @gray-50;
```

Comment:
That's true. I think we do want this to be self-contained, so even if the chance is slight, this can change, but it should be set to the global rule, not the color directly. Do we currently have a global rule variable color set?